### PR TITLE
Use antsibull-docs-parser for markup processing

### PR DESCRIPTION
**Pull request by felixfontein**
Another for #104.

### CODE REVIEW

**Comment by felixfontein** [comment]
Restart tests.

File: README.md [hunk 1]
```diff
@@ -71,17 +71,17 @@ and install the requirements needed to run the tests there.
...
+`../antsibull-docs-parser` when running tests if those paths exist.
 You can change this behavior through the `OTHER_ANTSIBULL_MODE` env var:
 
 - `OTHER_ANTSIBULL_MODE=auto` — the default behavior described above
-- `OTHER_ANTSIBULL_MODE=local` — install the project from `../antsibull-core`.
-  Fail if that paths doesn't exist.
+- `OTHER_ANTSIBULL_MODE=local` — install the projects from `../antsibull-core`
+  and `../antsibull-docs-parser`. Fail if that paths doesn't exist.
 - `OTHER_ANTSIBULL_MODE=git` — install the projects from the Github main branch
 - `OTHER_ANTSIBULL_MODE=pypi` — install the latest version from PyPI
```
Comment (gotmax23) [inline README.md:86]:
```suggestion
- `OTHER_ANTSIBULL_MODE=pypi` — install the latest versions from PyPI
```

File: .github/workflows/antsibull-docs.yml [hunk 2]
```diff
@@ -51,6 +53,13 @@ jobs:
           path: antsibull-core
           ref: ${{ matrix.antsibull_core_ref }}
 
+      - name: Check out dependent project antsibull-docs-parser
```
[the remark this answers is not in the document]
Comment (felixfontein) [reply .github/workflows/antsibull-docs.yml:56]: Thanks for spotting this! There were also two more places like this below, and in both places antsibull-docs-parser wasn't even checked out.

**Review by gotmax23 — APPROVED** [review]
:tada:

**Comment by felixfontein** [comment]
@gotmax23 thanks for reviewing this!

**Pull request merged**